Backends/RasterImage/output-destination.lisp

```diff
@@ -0,0 +1,58 @@
+;;;;  (c) copyright 2019 Jan Moringen
+
+;;;; This library is free software; you can redistribute it and/or
+;;;; modify it under the terms of the GNU Library General Public
+;;;; License as published by the Free Software Foundation; either
+;;;; version 2 of the License, or (at your option) any later version.
+;;;;
+;;;; This library is distributed in the hope that it will be useful,
+;;;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;;;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+;;;; Library General Public License for more details.
+;;;;
+;;;; You should have received a copy of the GNU Library General Public
+;;;; License along with this library; if not, write to the
+;;;; Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+;;;; Boston, MA  02111-1307  USA.
+
+(cl:in-package #:mcclim-raster-image)
+
+(defclass raster-image-destination (file-destination)
+  ((%file-format :initarg :file-format
+                 :reader file-format)
+   (climi::element-type :allocation :class
```

Comment:
ditto, this symbol if meant for backends should be exported from climb package

Reply:
Replaced `:allocation :class` with methods on `destination-element-type` which is already exported.